samples/event_manager_profiler/prj.conf

```diff
@@ -0,0 +1,23 @@
+#
+# Copyright (c) 2021 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+# Enabling assert
+CONFIG_ASSERT=y
+
+# Logger configuration
+CONFIG_LOG=y
+CONFIG_LOG_DEFAULT_LEVEL=2
+CONFIG_LOG_BACKEND_UART=y
+CONFIG_LOG_STRDUP_BUF_COUNT=10
+
+# Configuration required by Event Manager
+CONFIG_EVENT_MANAGER=y
+CONFIG_LINKER_ORPHAN_SECTION_PLACE=y
+CONFIG_HEAP_MEM_POOL_SIZE=2048
+CONFIG_REBOOT=y
+
+# Configuration required by Event Manager Profiler Tracer
```

Comment:
`# Configuration required by Event Manager profiler tracer`
Please pay attention to capitalization